service/src/main/resources/application.properties

```diff
@@ -9,6 +9,6 @@ spring.datasource.driver-class-name=org.postgresql.Driver
 
 # Logging
 logging.level.io.provenance.timed=warn
-#logging.level.Exposed=DEBUG
+logging.level.Exposed=ERROR
```

Comment:
Yes I know this will mask WARN messages from Postgres. I really only care about ERRORs, and if theres an exception, Im already catching it